src/main/java/ja/workshops/spring/części/konfiguracja/adnotacjami/a_adnotacje_wstrzykiwania/required/Samochód.java

```diff
@@ -0,0 +1,38 @@
+package ja.workshops.spring.części.konfiguracja.adnotacjami.a_adnotacje_wstrzykiwania.required;
+
+
+import org.springframework.beans.factory.annotation.Required;
+
+/**
+ * Przedstawienie działania adnotacji {@link Required}.
+ * <p>
+ * Od wersji 5.1 adnotacja @Required jest adnotacją przestarzałą z racji na to,
+ * że wymagane zależności powinny być wstrzykiwane przez konstruktor.
+ * <p>
+ * Do prezentacji działania adnotacji Required zmienić wersję Springa na 5.0.9 w pliku <code>pom.xml</code>.
+ *
+ * @author Wiktor Rup
+ */
+class Samochód {
+
+    private Integer rokProdukcji;
+    private String marka;
+
+    Integer getRokProdukcji() {
+        return rokProdukcji;
+    }
+
+    @Required
+    public void setRokProdukcji(Integer rokProdukcji) {
```

Comment:
Jakieś słowo o tym w jakiej fazie cyklu to będzie wołane oraz odnośniki: do kawałka kodu Springa, który to realizuje oraz do prezki o cyklu życia ziaren Springowych tu poproszę. Ten drugi odnośnik może być robiony przez osobę robiącą tamten kawałek materiału.